.github/workflows/aws-nuke/aws-nuke-ci.yaml

```
@@ -0,0 +1,23 @@
+name: Run aws-nuke on CI account.
+
+on:
+  schedule:
+    - cron: '0 7 * * *' # everyday at 1 AM Mountain
+
+jobs:
+  aws-nuke:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_COMMERCIAL_ROLE_TO_ASSUME }}
+          role-session-name: ${{ github.job || github.event.client_payload.pull_request.head.sha || github.sha }}
+          aws-region: us-west-2
+          role-duration-seconds: 21600
+
+      - name: Download aws-nuke
+        run: wget -c https://github.com/rebuy-de/aws-nuke/releases/download/v2.23.0/aws-nuke-v2.23.0-linux-amd64.tar.gz -O - | tar xz
+
+      - name: Run aws-nuke
+        run: ./aws-nuke-v2.23.0-linux-amd64 --config ./aws-nuke-config.yaml
```

According the [`aws-nuke` docs](https://github.com/rebuy-de/aws-nuke#caution), the `--no-dry-run` flag is required for it to actually delete resources. Maybe we want to have a "dry-run" stage and a "live" stage?

```suggestion
        run: ./aws-nuke-v2.23.0-linux-amd64 --config ./aws-nuke-config.yaml --no-dry-run
```

---

How would the dry-run and live stages work here?  Also, I suggest merging this without `--no-dry-run` for now to see it run on its own first and then enabling it with a follow-on PR.

---

I was thinking the dry-run stage would just be a step in the job prior to running the next step that uses the `--no-dry-run` flag to actually delete the resources. Similar in concept to running `terraform plan` prior to `terraform apply`. Not sure whether that scenario would be helpful for us or not in this use case. Moving forward without `--no-dry-run` sounds good to me.